src/PsfFlux.cc

```diff
@@ -110,10 +113,15 @@ void PsfFluxAlgorithm::measure(afw::table::SourceRecord& measRecord,
     // variance as if we had, so we'll apply the weights to the model now, and update alpha.
     result.instFluxErr = std::sqrt(model.array().square().matrix().dot(variance.cast<PsfPixel>())) / alpha;
     measRecord.set(_areaKey, model.sum() / alpha);
+    measRecord.set(_npixelsKey, fitRegion.getSpans()->getArea());
     if (!std::isfinite(result.instFlux) || !std::isfinite(result.instFluxErr)) {
         throw LSST_EXCEPT(PixelValueError, "Invalid pixel value detected in image.");
     }
     measRecord.set(_instFluxResultKey, result);
+    auto chi2 = ((data.cast<PsfPixel>() - result.instFlux * model).array().square() *
+                 variance.cast<PsfPixel>().array().rsqrt())
```

Comment:
This isn't right: we want the variance (sigma squared) in the denominator.

```suggestion
                 variance.cast<PsfPixel>().array())
```

Reply:
D'oh! I'm embarrassed I missed that. 🤦